client/src/components/FileTransfer/index.js

```diff
@@ -102,7 +102,7 @@ export default class FileTransfer extends Component {
     }
     return (
       <div className={`${styles} icon file-transfer btn btn-link`}>
-        <input type="file" name="fileUploader" id="fileInput" ref={c => this._fileInput = c} />
+        <input placeholder="Choose a file..." type="file" name="fileUploader" id="fileInput" ref={c => this._fileInput = c} />
```

Comment:
Would be good to translate, but not a blocker

Reply:
This string is not displayed unless you can't see the images, so it's an accessibility issue but it's a little better than when there was nothing. Can I translate it later in the #129 PR ?